iOSVideoApp/iOSVideoApp/Models/RecordingObject.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  RecordingObject.swift
+//  iOSVideoApp
+//
+//  Created by Ezra Black on 5/6/20.
+//  Copyright © 2020 Casanova Studios. All rights reserved.
+//
+
+import Foundation
+
+struct Recording {
+    var title: String
+    var url: URL //this should end up being the file output path
+}
```

Comment:
Nice & simple! Maybe to make that comment less necessary, call it `fileURL`?

Reply:
That’s a good idea I did not think about it!